src/powershell/Private/Get-HubIdentifier.ps1

```diff
@@ -12,33 +12,34 @@
 
     Returns the string '123456hyfpqje'.
 #>
-function Get-HubIdentifier
+function Get-HubIdentifier 
 {
-    param
-    (
+    param (
```

Comment:
Are you editing in VS Code with the recommended extensions? If so, can you confirm you don't have any personal preferences that are overriding the code style rules in the workspace? While we're totally open to changing code style, we want to keep it consistent to minimize unnecessary change back and forth between different styles. Most of the style rules should be applied automatically on save. 
```suggestion
    param
    (
```

Reply:
I need to check that. Do we have a template so that next time I commit ps code it is not deviating from the current style? I don't have a personal preference. It is all good for me.

Reply:
Do you want me to commit all suggestions per suggestion?

Reply:
Yes. I need to check my settings. Do we have any guide or template? I am okay with whatever style we need to adhere to. Do you want me to commit the suggestions? I seenitbis per suggestion

Reply:
I went ahead and opened the PR branch and just saved the files to auto-apply the current settings. There are a set of extensions that do this automatically. I think the PowerShell extension is the one working here. I just realized that isn't configured as a recommended extension, tho. I'll add that to this PR.
![image](https://github.com/user-attachments/assets/41814794-9949-4aea-855e-6cf8dfb7f37c)